src/CMakeLists.txt

```diff
@@ -310,8 +310,8 @@ target_link_libraries(migraphx PRIVATE nlohmann_json::nlohmann_json)
 find_package(SQLite3 REQUIRED)
 target_link_libraries(migraphx PRIVATE SQLite::SQLite3)
 
-find_package(msgpackc-cxx QUIET)
-if(NOT msgpackc-cxx_FOUND)
+find_package(msgpack-cxx QUIET)
```

Comment:
MIGraphX msgpack is pinned at 3.3.0
https://github.com/ROCm/AMDMIGraphX/blob/05c9d1ba5467e695d0b52839f5e95b8240fdb48a/requirements.txt#L28

6.0.0 API is not tested with MIGraphX in case there is any incompatibilities.

Reply:
That is understood. By API I meant in context of the cmake find_package API.

Reply:
Thanks for that link @umangyadav!